### DISCUSSION TITLE
Seeing private channels I dont have access to?

### DISCUSSION THREAD

**Discussion by winterfa11** [Tyrrrz/DiscordChatExporter#1281] [Q&A] [answered]
Hello. 
I'm seeing private channels I don't have access to, like admin channels. Is that actually something I can export and see as someone who wasn't given access? It'd be good to know if permissions were really that easy to bypass...

**Comment by 96-LB** [comment, marked as answer]
You can view the full list of channels through the Discord API, regardless of if you can actually access those channels. However, you are not able to bypass permissions and read the messages, so exporting them is not possible. If you tried with DiscordChatExporter you would get an "Access is forbidden" message.

**Reply by winterfa11** [reply]
Ah okay. Good to know, thanks!
